task-generator/remote/main.go

```diff
@@ -142,55 +144,65 @@ if [ -e "/ssh/error" ]; then
   #no server could be provisioned
   cat /ssh/error
   exit 1
+fi
+export SSH_HOST=$(cat /ssh/host)
+[ "$SSH_HOST" == "localhost" ] && IS_LOCALHOST=true
```

Comment:
nitpick: just if condition would be easier to read than a functional style

Reply:
I have changed this.